requirements.txt

```diff
@@ -10,4 +10,5 @@ erddapy>=0.6
 fsspec>=0.7.4
```

Comment:
Should we `fsspec>=0.7.4, <0.8.4` here to prevent the caching FileNotFound error ?

Reply:
yes, sounds good